mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -104,8 +107,8 @@ private[classification] trait LogisticRegressionParams extends ProbabilisticClas
    * @group setParam
    */
   def setThresholds(value: Array[Double]): this.type = {
-    if (isSet(threshold)) clear(threshold)
-    set(thresholds, value)
+    logWarning("Ignoring setThresholds(), use setThreshold() for binary Logistic Regression.")
```

Comment:
If we're going to include this patch, it makes most sense to me to remove the note entirely. When this functionality is restored, we can add back the note. From what I've seen, comments should not reference future "plans" since plans like that often change. JMO.